orders/tests.py

```diff
@@ -52,3 +52,14 @@ def test_dog_owner_id_is_deleted_when_dog_owner_is_deleted(self, create_order):
     def test_daycare_id_is_deleted_when_daycare_is_deleted(self, create_order):
         DayCare.objects.get(id=1).delete()
         assert Order.objects.get(id=create_order.id).daycare_id is None
+
+    def test_get_all_day_cares_available_on_dates_function(self):
+        start_date = "2022-05-02"
+        end_date = "2022-08-02"
+        day_care = DayCare.create(username='testuser', email='test@email.com', password='test_password',
+                                  name='name', description='test description', price_per_day=500,
+                                  capacity=1, area='Center', city='Tel_aviv', address='address')
+        assert day_care in Order.get_all_day_cares_available_on_dates(start_date, end_date)
+        Order.create(dog_owner_id=DogOwner.objects.get(id=1), daycare_id=day_care,
+                     start_date="2022-05-02", end_date="2022-08-02", price_per_day=500).approve_order()
+        assert day_care not in Order.get_all_day_cares_available_on_dates(start_date, end_date)
```

Comment:
This doesn't really belong here. instead, you should add a test that ensure a specific daycare with an order created for it is not found by the function.

Reply:
no. the test is named `test_get_all_day_cares_available_on_dates_function`.
a `not in` assertion test something completely different - it tests that a daycare is *not available*

Reply:
Got your point.